packages/create-amplify/src/amplify_project_creator.ts

```diff
@@ -14,7 +14,7 @@ export class AmplifyProjectCreator {
   private readonly defaultDevPackages = [
     '@aws-amplify/backend',
     '@aws-amplify/backend-cli',
-    'typescript@^5.0.0',
+    'typescript@^5.0.0', // TODO: remove this line for yarn since it's installed in amplify/
```

Comment:
is there any change here ?

Reply:
I removed `typescript` from `defaultDevPackages` for `yarn` and `yarn-stable` because they need to be installed inside `amplify/`

Reply:
checked. `amplify` folder can't consume the package from its parent folder. It might work if we make it as a workspace, but I don't think we want to do that, do we?